--- distinfo.orig
+++ distinfo
@@ -49,3 +49,9 @@
 SIZE (rust/2026-04-16/rust-std-1.95.0-riscv64gc-unknown-freebsd.tar.xz) = 21496384
 SHA256 (rust/2026-04-16/cargo-1.95.0-riscv64gc-unknown-freebsd.tar.xz) = 3403e0ff5ccc20d8e79f2025f540929fc5a2165d6e2ef6db18282c83f4364193
 SIZE (rust/2026-04-16/cargo-1.95.0-riscv64gc-unknown-freebsd.tar.xz) = 10903448
+SHA256 (rust/2026-04-16/rustc-1.95.0-x86_64-unknown-dragonfly.tar.xz) = 420fd1c1b0e8783cb2ed0c6d0442165f6685e6f4299d46222a90de74bccfa9b6
+SIZE (rust/2026-04-16/rustc-1.95.0-x86_64-unknown-dragonfly.tar.xz) = 67952936
+SHA256 (rust/2026-04-16/rust-std-1.95.0-x86_64-unknown-dragonfly.tar.xz) = 7fa1435ae2d6ffabdc205ed012f36a7c8e487630862d64e51dffe79bbee82ea7
+SIZE (rust/2026-04-16/rust-std-1.95.0-x86_64-unknown-dragonfly.tar.xz) = 24176808
+SHA256 (rust/2026-04-16/cargo-1.95.0-x86_64-unknown-dragonfly.tar.xz) = 3ca19cc016d70958d385183fc657a19e8c29528245742cee75f37386ba73ee5a
+SIZE (rust/2026-04-16/cargo-1.95.0-x86_64-unknown-dragonfly.tar.xz) = 11307304
